### PR TITLE
Adding a file for Derry, Northern Ireland

### LINKED ISSUE #3: ciaran036 Recommendations
## Things to do in Derry/Londonderry, Northern Ireland

Let's add a file for people who are visiting Derry/Londonderry for the first time! 

### Steps for this project: 
- [x] Create a branch
- [x] Create a file
- [x] Commit my changes
- [x] Open a pull request
- [ ] Request a review
- [ ] Make changes as needed
- [ ] Get an Approval
- [ ] Merge the Pull Request

### PR DESCRIPTION
**Pull request by ciaran036**
This Pull Request resolves #3 by adding some recommendations for places to eat and things to do in Derry/Londonderry, Northern Ireland. 

Something something something. 

Please provide more information on the types of food you like and the things you enjoy doing when you travel! @paulbreen

### CODE REVIEW

**Comment by paulbreen** [comment]
Derry

File: ciaran036-derry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Derry/Londonderry, Northern Ireland
+
+### Great places to eat
+
```
Comment (ciaran036) [inline ciaran036-derry.md]: lethal

Comment (paulbreen) [inline ciaran036-derry.md]: Sort it out

File: ciaran036-derry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Derry/Londonderry, Northern Ireland
```
Comment (paulbreen) [inline ciaran036-derry.md]: Derry, Northern Ireland

**Comment by ciaran036** [comment]
@paulbreen please review?

File: ciaran036-derry.md (hunk 1 above)
Comment (David-Harvey) [inline ciaran036-derry.md]: Can I have some more please?

**Comment by David-Harvey** [comment]
I have since approved this as it clustered with comments and I'm confused :smile:

**Comment by ciaran036** [comment]
@David-Harvey Changes made, please review?

**Pull request merged**